backend/src/test/java/com/huellapositiva/api/ProposalControllerShould.java

```diff
@@ -609,4 +608,79 @@ void list_every_existing_proposal_that_is_not_inadequate_as_reviser() throws Exc
 
         assertThat(listedProposals.getProposals().size()).isEqualTo(2);
     }
+
+
+    @Test
+    public void return_200_and_a_list_of_volunteers_from_a_proposal_as_reviser() throws Exception {
+
+        // GIVEN
+        String proposalId = testData.registerESALAndProposalWithInscribedVolunteers().getId();
+        testData.createCredential("revisor@huellapositiva.com", UUID.randomUUID(), DEFAULT_PASSWORD, Roles.REVISER);
+        JwtResponseDto jwtResponseDto = loginAndGetJwtTokens(mvc, "revisor@huellapositiva.com", DEFAULT_PASSWORD);
+
+        MockHttpServletResponse fetchResponse = mvc.perform(get(FETCH_PROPOSAL_URI + proposalId + "/volunteers")
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + jwtResponseDto.getAccessToken())
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn().getResponse();
+
+
+        ArrayList<VolunteerDto> listedVolunteers = objectMapper.readValue(fetchResponse.getContentAsString(), ArrayList.class);
+        assertThat(listedVolunteers.size()).isEqualTo(1);
+
+    }
+
+    @Test
+    public void return_400_when_proposal_not_found() throws Exception{
```

Comment:
When NOT_FOUND we would probably like to return 404

Reply:
Hemos cambiado el controlador, porque se esperaba que si no encontraba la proposal, lanzara una excepción con el error 400, en vez del 404 que es NOT_FOUND.